docs/build/cadence/advanced-concepts/scheduled-transactions.md

```diff
@@ -0,0 +1,483 @@
+---
+title: Scheduled Transactions
+description: Learn about Flow Scheduled Transactions, enabling smart contracts
+to autonomously execute predefined logic at specific future times without external triggers.
+keywords:
+  - scheduled transactions
+  - autonomous execution
+  - Flow blockchain
+  - smart contracts
+  - transaction scheduling
+  - FlowTransactionScheduler
+  - time-based automation
+  - transaction handlers
+  - blockchain automation
+sidebar_position: 8
+---
+
+# Flow Scheduled Transactions Documentation
+
+## Introduction
+
+::::warning
+Scheduled tranasctions are part of the Forte network upgrade and are currently available on Flow Emulator (CLI v2.7.0+) and [Flow Testnet]. See the announcement for context: [Forte: Introducing Actions & Agents].
+::::
+
+Scheduled transactions on the Flow blockchain enable users and smart contracts to autonomously execute predefined logic at specific future times without external triggers. This powerful feature allows developers to create "wake up" patterns where contracts can schedule themselves to run at predetermined block timestamps, enabling novel blockchain automation patterns.
+
+Key benefits include:
+- **Autonomous execution**: No need for external services or manual intervention
+- **Time-based automation**: Execute transactions based on blockchain time
+- **Predictable scheduling**: Guaranteed execution within specified time windows  
+
+Common use cases include recurring payments, automated arbitrage, time-based contract logic, delayed executions, and periodic maintenance tasks.
+
+::::info
+Flow provides a scheduled transaction manager to make managing your scheduled transactions more streamlined. Check out the [scheduled transactions intro](../../../blockchain-development-tutorials/forte/scheduled-transactions/scheduled-transactions-introduction.md) for a tutorial on how to schedule some basic transactions with the manager.
+::::
+
+## Concepts
+
+### Creating a Scheduled Transaction
+
+In order to create a scheduled transaction, the logic that will be executed
+in the transaction must already be defined in a function that the scheduler
+will call when it is time for the transaction to be executed.
+
+Therefore, all scheduled transactions must include a capability to a resource that conforms to this Transaction Handler interface defined in the Scheduler contract
+and includes getters that conform to the [Flow metadata views standard](./metadata-views.md):
```

Comment:
How come we added this examples here, if we have a section of examples bellow. I just wonder if it will be confusing for someone new to read this code before even knowing the concepts of scheduling transactions.

Reply:
I thought it would be useful to have a relevant example of scheduled transaction logic so users know what is actually being scheduled in later sections, but I understand that it might be too much. @briandoyle81 what do you think?